addons/website_slides/i18n/website_slides.pot

```diff
@@ -6509,3 +6509,9 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:website_slides.course_nav
 msgid "└<span class=\"ms-1\">Uncategorized</span>"
 msgstr ""
+
```

Comment:
The translation is not added in the right place in the pot file, you need to follow these steps:
Settings ‣ Translations ‣ Import / Export ‣ Export Translations ‣ eLearning/pot
find the position of the msg you added on the python side in the download file, then edit the file and add it.
In your case, it should be added in the line 6073

Reply:
https://github.com/odoo/odoo/wiki/Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions